apps/server/src/shared/domain/entity/boardnode/types/board-node-type.ts

```diff
@@ -3,4 +3,5 @@ export enum BoardNodeType {
 	COLUMN = 'column',
 	CARD = 'card',
 	TEXT_ELEMENT = 'text-element',
+	FILE_ELEMENT = 'file-element',
```

Comment:
I think the domain should managed for composits what can be exits (AnyContentElementDo, ContentElementType, ..). 
It exists no need for a additional representation on datarecord side. 
Can you please make a look with north* on this topic?

Reply:
its actually the other way around, the domain has no need to somehow list what types exist, each composite is a composite of a specific subclass, and its type is known from the subclass, and any details specific to the subclass can be handled by it.

The datalayer on the other hand needs to know what type of composite to instanciate when it loads it from database. For this the type needs to be stored, and for that it needs an enum.

Reply:
AnyBoardDo and similar are types, they cant be used for this distinction.